src/frontend/screens/Settings/MapManagement/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {type NativeStackNavigationOptions} from '@react-navigation/native-stack';
+import {defineMessages, useIntl, type MessageDescriptor} from 'react-intl';
+import {ScrollView} from 'react-native';
+
+import {List, ListItem, ListItemText} from '../../../sharedComponents/List';
+import {type NativeRootNavigationProps} from '../../../sharedTypes/navigation';
+
+const m = defineMessages({
+  screenTitle: {
+    id: 'screens.Settings.MapManagement.screenTitle',
+    defaultMessage: 'Map Management',
+  },
+  backgroundMaps: {
+    id: 'screens.Settings.MapManagement.backgroundMaps',
+    defaultMessage: 'Background Maps',
+  },
+});
+
+export function MapManagementScreen({
+  navigation,
+}: NativeRootNavigationProps<'MapManagement'>) {
+  const {formatMessage: t} = useIntl();
+  return (
+    <ScrollView>
+      <List>
+        <ListItem
+          onPress={() => {
+            navigation.navigate('BackgroundMaps');
+          }}>
+          <ListItemText primary={t(m.backgroundMaps)} />
+        </ListItem>
+      </List>
+    </ScrollView>
```

Comment:
[This should have a top padding of 40](https://www.notion.so/digidem/Spacing-74d09a0bdefa4bc5b9508336f7b2142a?pvs=4#515a2f0e66d944348c7dd350df02cf6c)

Reply:
seems somewhat questionable for that design spec to apply to this page, especially when compared to other settings-like pages.

App settings:

<img width="200" alt="image" src="https://github.com/user-attachments/assets/ff8184bf-872c-4836-8113-030fecd47e16">

Screen in question (with your suggestion):

<img width="200" alt="image" src="https://github.com/user-attachments/assets/fa9d0531-8b0d-4e08-a42a-ace59384cd16">

My guess is that either:

- the app settings page needs to be updated too
- the design spec does not apply to pages like these

Reply:
> the app settings page needs to be updated too

ah yeah it this seems likely when looking at the screenshot in https://github.com/digidem/comapeo-mobile/issues/622

I think for the sake of consistency, I'm going to avoid making this change so that it matches the app settings layout. we can address both of those in a follow up (since there are a good number of design changes needed anyways)